tizen/src/platform_tizen.cpp

```diff
@@ -220,9 +226,103 @@ std::string stringFromFile(const char* _path) {
     return out;
 }
 
-// No system fonts fallback implementation (yet!)
+void initPlatformFontSetup() {
```

Comment:
Do you think we could make that a lazy initialization on tizen? It seems pretty inconvenient to add this to all other platforms.